### DISCUSSION TITLE
Vanilla gnome spiral Linux

### DISCUSSION THREAD

**Discussion by antonispgs** [SpiralLinux/SpiralLinux-project#482] [Q&A] [answered]
I appreciate the effort begind spiral and understand the purpose of that installer, especially the rollback system and btrfs layout but I would prefer if everything else was vanilla gnome install on Debian. Is there a quick way to do that? Is it easy to get rid of duplicate gnome software for example or enabling Wayland by default and removing individual apps, extensions etc or does it all have to be done manually?

**Comment by geckolinux** [comment, marked as answer]
Hi there. Yep, you can have a vanilla Gnome (per Debian's defaults) by deleting the contents of `/etc/skel/*` and also the dotfiles in your `/home/` director(y)(ies).


The two entries for Gnome Software are not duplicates, the **(root)** entry is so that you can administer multiple system packages without it bothering you for the root password for each one. If you still want to remove it then delete `/usr/share/applications/gnome-software-root.desktop` 

Regarding Wayland: https://github.com/orgs/SpiralLinux/discussions/316#discussioncomment-7215164

**Comment by techarmy01** [comment]
Hello Dev @geckolinux 

I was also trying to get vanilla gnome without extension and deleting `/etc/skel/*` didn't get work for new user profile.  

I still get Spiral Linux version where dash dock, themes and other mods were still when I created a new user. 

Strange...

Any other files to remove ?

Thanks.

**Reply by geckolinux** [reply]
Hi there, you would have to delete `/etc/skel/*` _before_ creating the new user account.
If the new user account is already created then you can delete `~/.config/*` (assuming you don't have other program settings there yet that you want to keep).

**Reply by techarmy01** [reply]
Got it. Thanks, will try it 😃
